docs/cli.rst

```diff
@@ -1,11 +1,271 @@
+
+After installing CumulusCI, use the ``cci`` command in your terminal or command prompt to interact with it.
+
+On a macOS, access the terminal via Terminal.app. On Windows, open cmd.exe.
+
+To see all available commands, type ``cci`` in your terminal:
+
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To get information on a specific command listed above, type ``cci <command>``. 
+
+    For example, if you would like more information on the ``task`` command, type ``cci task``:
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+There are even more subcommands available under ``cci task``.
+
+
+
+Getting ``--help``
+------------------
+
+If you want help running a task, type ``cci task run --help``. If you're not certain about what a specific command does, use the ``--help`` option to get more information. 
+
+.. code-block:: console
+
+    $ cci task run --help
+    Usage: cci task run [OPTIONS] TASK_NAME
+
+    Runs a task
+
+    Options:
+    --org TEXT      Specify the target org.  By default, runs against the
+                    current default org
+
+    -o TEXT...      Pass task specific options for the task as '-o option
+                    value'.  You can specify more than one option by using -o
+                    more than once.
+
+    --debug         Drops into pdb, the Python debugger, on an exception
+    --debug-before  Drops into the Python debugger right before task start.
+    --debug-after   Drops into the Python debugger at task completion.
+    --no-prompt     Disables all prompts.  Set for non-interactive mode use such
+                    as calling from scripts or CI systems
+
+    --help          Show this message and exit.    
+
+This gives us a clear usage statement, description, and a list of all options available for use with the command.
```

Comment:
```suggestion
This gives us a clear usage statement, description, and a list of all options available with the command.
```

Reply:
How about avoid the pronoun entirely and say "The ``--help`` option gives us..."